### DISCUSSION TITLE
Tidal Developer Portal API Dart client

### DISCUSSION THREAD

**Discussion by filippomenchini** [tidal-music/discussions#19] [General]
Hi everyone!

I'm very happy to see Tidal finally opening up to the developer community, can't wait to see the next updates!

In the meantime, I started developing a Dart library to connect to and use APIs easily.
I hope you like it, feedback is always welcome.

Obviously it's still in a primordial state, I started working on it just yesterday.

Here's the repo: https://github.com/filippomenchini/tidal

**Comment by stefanmoro** [comment]
Hey @filippomenchini, and welcome to the forum!

Amazing! It is really great to see people starting to contribute to the TIDAL Developer Platform! Even though we hope to gradually open up more of our platform, contributions like this are important in order for TIDAL Developer Platform to be easily accessible from different platforms and programming languages. 

Thanks!

**Reply by filippomenchini** [reply]
Hello @stefanmoro, thanks for your feedback!

I'll keep you posted on the development of this library.
I'm currently working on the Album APIs, Artist APIs are already implemented.

Hope to release to pub.dev soon!

**Comment by filippomenchini** [comment]
Hello everyone!

I've finally managed to complete the first version of the Tidal API Dart client!
Here's the package page: https://pub.dev/packages/tidal
This is still the first version, it gets the job done and I did my best to test this and to make it readable, every feedback is welcome!

**Reply by aliaksei-taliuk** [reply]
@filippomenchini hey!

Thanks a lot for your contribution! We will make an internal announcement if someone within TIDAL engineering need to rely on Dart SDK in the future. Please let us know if you have any further questions, we will be happy to help you. 

Great job though!

**Reply by filippomenchini** [reply]
Thank you for your feedback!
I'll keep you posted on the development
